modules/programs/khal.nix

```diff
@@ -29,7 +29,6 @@ let
       "path = ${
         value.local.path + "/"
         + (optionalString (value.khal.type == "discover") value.khal.glob)
-        + "/*"
```

Comment:
This will be added from `khal.glob` in case `type` is set to `discover`.
And other type `calendar` doesn't have globbing, but only the path.

Reply:
I fixed my setup with this change and that fixed it ty.